exporter/kafkaexporter/README.md

```diff
@@ -1,11 +1,15 @@
 # Kafka Exporter
 
+| Status                   |                      |
+| ------------------------ |----------------------|
+| Stability                | [beta]               |
```

Comment:
@pavolloffay, @MovieStoreGuy, as code owners of this component, do you agree with this [stability level](https://github.com/open-telemetry/opentelemetry-collector#stability-levels)?